src/test/unit/common/services/downloadService.test.ts

```diff
@@ -69,9 +61,9 @@ suite('DownloadService', () => {
   test('Tries to download LS if not installed', async () => {
     configuration = {
       isAutomaticDependencyManagementEnabled: () => true,
-      getCustomCliPath: () => undefined,
-      getSnykLanguageServerPath: () => 'abc/d',
-    } as unknown as IConfiguration;
+      getCliReleaseChannel: () => 'stable',
+      getCliPath: () => Promise.resolve('path/to/cli'),
```

Comment:
we should have a test that checks usage of the default value - maybe it's there, but haven't seen it yet.

Reply:
test in configuration.tests.ts